contrib/phase_distribution_function.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
Can you add a line that says what the file does and move it to [contrib/grain_size/phase_distribution_function.ipynb](https://github.com/grain-damage-Mulyukova/aspect/pull/5/files#diff-dc46a07843680930bcb01e7514aef17cebbb27d6154ea9c8d07810abe5cc8aed)